File: OnlineGroceryOrderingSystemAuthenticationJava/src/main/java/com/alpha/ogo/security/AuthenticationResponse.java
package com.alpha.ogo.security;

import java.io.Serializable;
import java.util.Collection;

import org.springframework.security.core.GrantedAuthority;

import lombok.Data;

@Data
public class AuthenticationResponse implements Serializable {

    private final String token;
    private final Long userId;
    private final String userName;
    private final boolean active;
    private final Collection<? extends GrantedAuthority> roles;

	private final String email;
	private final String fullName;


    public AuthenticationResponse(MyUserDetails userDetails,String jwt) {
		super();
		this.token = jwt;
		this.userId = userDetails.getUserId();
		this.userName = userDetails.getUsername();
		this.active = userDetails.isEnabled();
		this.roles = userDetails.getAuthorities();
		this.email = userDetails.getEmail();
		this.fullName = userDetails.getFullName();
	}

	/*
	 * public String getJwt() { return token; }
	 */
}

File: OnlineGroceryOrderingSystemAuthentication/src/main/kotlin/com/allion/ogoss/security/AuthenticationResponse.kt
package com.allion.ogoss.security

import com.allion.ogoss.domain.User
import org.springframework.security.core.GrantedAuthority

class AuthenticationResponse(var token: String?, user: User, role: Collection<GrantedAuthority?>?) {
    var userId: Long? = null
    var userName: String? = null
    var active = false
    var roles: Collection<GrantedAuthority?>? = role
    var email: String? = null
    var fullName: String? = null

    init {
        this.userId=user.userId
        this.userName=user.phoneNo
        this.active=user.active
        this.email =user.email
        this.fullName = user.fullName
    }
}